src/viking.js

```diff
@@ -1,11 +1,82 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health = health;
+        this.strength =  strength;
+    }
+    attack() {
+        return this.strength;
+    }
+    receiveDamage(damage) {
+        let currentHealth = this.health - damage;
```

Comment:
You don't need a new variable here. We need to reference `this.health`

```suggestion
        this.health = this.health - damage;
```